include/okapi/api/control/async/asyncLinearMotionProfileController.hpp

```diff
@@ -267,5 +267,12 @@ class AsyncLinearMotionProfileController : public AsyncPositionController<std::s
 
   std::string
   getPathErrorMessage(const std::vector<Waypoint> &points, const std::string &ipathId, int length);
+
+  /**
+   * Attempts to remove a path without stopping execution, then if that fails, stops exection and removes the path
+   * Returns true if the path was successfully removed
+   * todo expose this?
```

Comment:
Nit: Make `todo` `TODO: ` because then CLion knows it's a todo and will highlight it in yellow and make them searchable